ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -151,69 +154,54 @@ extension UIColor {
     }
 
     @objc public var light: UIColor {
-        guard let color = resolvedColorValue(userInterfaceStyle: .light) else {
-            return self
-        }
-        return UIColor(colorValue: color)
+        let color = resolvedColorValue(userInterfaceStyle: .light)
+        return color
     }
 
     @objc public var lightHighContrast: UIColor {
-        guard let color = resolvedColorValue(userInterfaceStyle: .light,
-                                             accessibilityContrast: .high) else {
-            return self
-        }
-        return UIColor(colorValue: color)
+        let color = resolvedColorValue(userInterfaceStyle: .light,
```

Comment:
Would it make sense to remove `color` and just return the result of `resolvedColorValue()`?

Reply:
Ah I left that in for some ease of debugging while I was moving code around. Pretty sure it gets compiled out either way, but sure I'm happy to tighten it up a bit.